services/security_group_manager/src/main/java/com/futurewei/alcor/securitygroup/utils/RestParameterValidator.java

```diff
@@ -173,7 +173,7 @@ public static void checkProtocolAndEtherType(SecurityGroupRule securityGroupRule
     }
 
     public static void checkSecurityGroupRule(SecurityGroupRule securityGroupRule) throws Exception {
-        checkTenantId(securityGroupRule.getTenantId());
+        //checkTenantId(securityGroupRule.getTenantId());
```

Comment:
is tenant id in SG rule not required?

Reply:
Yes, tenant id not in SG rule request params.

Reply:
Cool thanks for the confirmation.